### PR TITLE
ReleaseNotes: Updated changelog and release notes for 8.1.3

### PR DESCRIPTION
**Pull request by grafanabot**
This exciting! So much has changed!

### CODE REVIEW

**Comment by marefr** [comment]
I think the two changes under `Features and enhancements` should be under bug fixes - I've added the `type/bug` to those PRs, but changes will reflect when re-generating the changelog.

**Comment by marefr** [comment]
Given all the changes in [milestone](https://github.com/grafana/grafana/milestone/253?closed=1) I would probably have expected more things in the changelog. But maybe non-related things except what's included now?

**Comment by sunker** [comment]
> Given all the changes in [milestone](https://github.com/grafana/grafana/milestone/253?closed=1) I would probably have expected more things in the changelog. But maybe non-related things except what's included now?

Yeah I think so too. Having said that, it must be up to the contributor/reviewer to add the label. 

> I think the two changes under `Features and enhancements` should be under bug fixes - I've added the `type/bug` to those PRs, but changes will reflect when re-generating the changelog.

Should I delete this branch and re-run the changelog action?

**Comment by marefr** [comment]
> Yeah I think so too. Having said that, it must be up to the contributor/reviewer to add the label.

Totally agree.
 
Should I delete this branch and re-run the changelog action?

Not sure, think we can make changes here given we've changed labels on the PRs so re-generating in the future should produce the same result. I can suggest changes

File: docs/sources/release-notes/release-notes-8-1-3.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+list = false
++++
+
+<!-- Auto generated by update changelog github action -->
+
+# Release notes for Grafana 8.1.3
+
+### Features and enhancements
+
+- **Bug:** Return empty plugins.DataTimeSeriesSlice when the frame field vector is empty and does not contain any data. [#38587](https://github.com/grafana/grafana/pull/38587), [@idafurjes](https://github.com/idafurjes)
```
Comment (marefr) [inline docs/sources/release-notes/release-notes-8-1-3.md:13]: Hmm, why is this not included in the changelog.md? Seems like a bug to me? Perhaps something to investigate later...

**Comment by marefr** [comment]
@sunker After consideration I would suggest closing this, deleting the branch and re-trigger the changelog generation.

**Comment by sunker** [comment]
Closing this so we can re-generate changelog

**Pull request closed without merging**